src/assets/scripts/client/aircraft/AircraftConflict.js

```diff
@@ -114,7 +114,6 @@ export default class AircraftConflict {
 
         this._recalculateLateralAndVerticalDistances();
         this.checkCollision();
-        this.checkRunwayCollision();
```

Comment:
Why is this removed?

Reply:
This checked to see if two aircraft were intending to use the same runway opposite direction. It's an unnecessary check that's outside the scope of a radar sim. All that matters is distance and direction, which are checked elsewhere. This was a very old method.